### ISSUE TITLE
Debugger - could not connect error

### ISSUE THREAD

**Issue by gsum5280** [pkulchenko/ZeroBraneStudio#384]
Running ZB IDE 0.90, Mobdebug 0.607 on a windows 7 machine and using Corona interpreter. I am trying to run the mobdebug script and have added the require statement in the main lua file.

I have been struggling with the following error: "Could not connect to localhost:8172". I read the other post and have tried the following:
1) Inserted the IP address of my computer in the start() statement.
2) Modified the cfg file to include a specific port with debugger.port = ' # '

Basically the problem comes down to the fact that it keeps attempting to use port 8172 and I can't seem to get the port number changed.

Just to clarify, When I choose Project | Start Debugging Server, it recognizes the port I specified in the cfg file. However when I try Project | Run, it reverts back to 8172.

So my question is how do I get ZB to recognize the port specified in the cfg when I choose Project | Run or Project | Start Debugging?

Thanks
